core/src/test/java/com/linecorp/armeria/common/util/CompositeExceptionTest.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.common.collect.ImmutableList;
+
+class CompositeExceptionTest {
+
+    private static final String separator = System.getProperty("line.separator");
+    private static final Sampler<Class<? extends Throwable>> alwaysVerboseExceptionSampler =
+            Sampler.always();
+    private static final Sampler<Class<? extends Throwable>> neverVerboseExceptionSampler =
+            Sampler.never();
+
+    @Test
+    void verboseExceptionEnabledTest() {
+        final IllegalStateException ex1 = new IllegalStateException();
+        final IllegalArgumentException ex2 = new IllegalArgumentException();
```

Comment:
How about also adding a test case that creates the composite exception with the exceptions that have cause?
```java
final IllegalArgumentException ex2 = new IllegalArgumentException();
final AnticipatedException ex3 = new AnticipatedException(ex2);
```
I think we need to also check if the padding is correctly appended because we have changed the code. 😉

Reply:
It seems necessary. I will add it. 😀